src/scorer.py

```diff
@@ -13,7 +13,7 @@
 class DiffAsmLine:
     line: str = attr.ib(cmp=False)
     mnemonic: str = attr.ib()
-    macro_arg: str = attr.ib()
+    macro_arg: str = attr.ib(cmp=False)
```

Comment:
should be possible to remove `macro_arg` completely now, actually, it was only there for SequenceMatcher's sake